content/news/2017-01-stats-post.md

```diff
@@ -0,0 +1,73 @@
+The current release has been out for almost three months now. Time to take a look at some statistics!
+
+At the time when this post was drafted, there have been a recorded 53953 downloads from the official OpenRA homepage. Not unsurprisingly, most downloaded packages are for the Windows platform (almost 75%!), with Linux being a distant second at 13% and OS X coming last with the remaining 12%. It is notable that the shares of both Linux and OS X are much larger than their respective share of the desktop market.
+
+When launching OpenRA for the first time, players get greeted by a question whether to send system information back to the OpenRA project &mdash; the very data which is used to compile these statistics! Of these more than 50k downloads, almost 22,000 individuals (thanks to everyone of you!) opted in to this data gathering, which comes out to about 40%.
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-downloads-vs-users.png)
+
+Linux users are a bit more likely to enable the sysinfo gathering than the average, OS X users less so.
```

Comment:
I removed the sentence. Also, here's another statistic:
![20170110-stats-mod-popularity-per-platform](https://cloud.githubusercontent.com/assets/4331210/21819111/a11bdf48-d76a-11e6-8a50-7005dff80ecb.png)
OS X users more likely to follow the mainstream! Shocking, I know. _/me runs_ ;)